pythonFiles/pytest-vscode-integration/tests/test_vscode_integration.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+
+
+def test_bar_fixture(testdir):
+    """Make sure that pytest accepts our fixture."""
+
+    # create a temporary pytest test module
```

Comment:
```suggestion
    # Create a temporary pytest test module.
```

All comments should be complete sentences.